src/Server.ts

```diff
@@ -1,27 +1,35 @@
+import * as bodyParser from 'body-parser';
 import * as  express from 'express';
-
+import { middleware2 } from './libs/routes/errorHandler';
+import { middleware1 } from './libs/routes/notFoundRoute';
 const app = express();
-
 export default class Server {
     constructor(private config) {
         this.setupRoutes();
         this.run();
         this.bootstrap();
+        this.initBodyParser();
 
     }
     public setupRoutes() {
-        // const {app}=this;
-        app.get( '/health-check' , (req , res ) => {
+       app.use( '/health-check' , (req , res ) => {
             res.send('I am ok');
         });
+       app.use(middleware1);
```

Comment:
@rohit-ns Please use proper names for middlewares, instead of middleware1 and middleware2, use meaningful names.